bluepill/tests/BPPackerTests.m

```diff
@@ -137,10 +182,10 @@ - (void)testPacking {
     XCTAssertEqual([bundles[1].skipTestIdentifiers count], 0);
     XCTAssertEqual([bundles[2].skipTestIdentifiers count], 0);
     XCTAssertEqual([bundles[3].skipTestIdentifiers count], 0);
-    XCTAssertEqual([bundles[4].skipTestIdentifiers count], 149);
-    XCTAssertEqual([bundles[5].skipTestIdentifiers count], 149);
-    XCTAssertEqual([bundles[6].skipTestIdentifiers count], 149);
-    XCTAssertEqual([bundles[7].skipTestIdentifiers count], 159);
+    XCTAssertEqual([bundles[4].skipTestIdentifiers count], 152);
+    XCTAssertEqual([bundles[5].skipTestIdentifiers count], 152);
+    XCTAssertEqual([bundles[6].skipTestIdentifiers count], 152);
+    XCTAssertEqual([bundles[7].skipTestIdentifiers count], 165);
```

Comment:
This is hard to maintain... we should probably do something like:

```
testsPerBundle = app.testBundles.count / numSims;
testsInFinalBundle = testsPerBundle + app.testBundles.count % numSims;
for (I = 0; I < bundles.count; ++i) {
  if (I < 4) XCTAssertEqual(bundles[I].skipTestIdentifiers.count, 0);
  else if (I < bundles.count -1) XCTAssertEqual(bundles[I].skipTestIdentifiers, testsPerBundle);
  else /* last bundle */ XCTAssertEqual(bundles[I].skipTestIdentifiers, testsInFinalBundle);
}
```

I know this bad pattern was here before, but since you're touching it anyway :-)

Reply:
Done. Replaced it with something similar.